packages/config/eslint-preset.js

```diff
@@ -0,0 +1,29 @@
+module.exports = {
+  env: {
+    browser: true,
+    es2021: true,
+  },
+  extends: [
+    'eslint:recommended',
+    'plugin:@typescript-eslint/eslint-recommended',
+    'plugin:@typescript-eslint/recommended',
+    'plugin:react/recommended',
+    'plugin:react-hooks/recommended',
+    'plugin:import/recommended',
+    'plugin:import/typescript',
+    'plugin:prettier/recommended',
+  ],
+  parser: '@typescript-eslint/parser',
+  parserOptions: {
+    ecmaFeatures: {
+      modules: true,
+      jsx: true,
+    },
+    ecmaVersion: 6,
+    sourceType: 'module',
+  },
+  plugins: ['prettier', 'jsx-a11y', 'react', '@typescript-eslint'],
+  rules: {
+    'react-hooks/exhaustive-deps': 0,
+  },
+};
```

Comment:
any particular reason to change this from what tsdx has to offer?

Reply:
tsdx is using older version of eslint (6.1.0) and if we provide custom implementation then same implementation can be used in all modules and apps. it will not be specific to tsdx modules

Reply:
cool